src/porepy/models/constitutive_laws.py

```diff
@@ -1697,6 +1711,8 @@ def potential_trace(
         bound_pressure_face_discr = dir_filter - (
             neu_filter + internal_boundary_filter
         ) * (one / t_f_full)
+        # On tip faces, the face contribution to the boundary potential is zero.
+        bound_pressure_face_discr = bound_pressure_face_discr * (one - tip_filter)
```

Comment:
Is this neccessary? If the filters are disjoint, I think this line should not impact the result

Reply:
The point is that bound_pressure_face_discr should be zero on tips, which this line enforces. Note that this is different from the other kinds of boundaries, where there is a contribution.

I can move 1715 a few lines up to multiply t_f_full instead if you think that is more transparent, but the filter must be applied somewhere.

Reply:
I'm missing something. Let's just talk in person.